src/item.h

```diff
@@ -2013,19 +2028,20 @@ class item : public visitable<item>
 
     public:
         int charges;
-
-        // The number of charges a recipe creates.  Used for comestible consumption.
-        int recipe_charges = 1;
-        int burnt = 0;           // How badly we're burnt
-        int poison = 0;          // How badly poisoned is it?
-        int frequency = 0;       // Radio frequency
-        int note = 0;            // Associated dynamic text snippet.
-        int irradiation = 0;      // Tracks radiation dosage.
-        int item_counter = 0; // generic counter to be used with item flags
+        units::energy energy;      // Amount of energy currently stored in a battery
+
+        int recipe_charges =
+            1;    // The number of charges a recipe creates.  Used for comestible consumption.
```

Comment:
astyle thinks this line is too long witht he comment like this.  Could you move it back to the previous line or otherwise tweak it?

Reply:
I had it on one line originally and this is what it changed to when i ran make astyle. Cut redundant information off the comment to shorten it.